ProcessingShowPath/Reader.pde

```diff
@@ -0,0 +1,65 @@
+class Reader{ 
+  private String[] filenames;
+  
+  Reader()
+  {
+    
+  }
+  
+  //get filenames from a specified folder and print them
+  void getFilesFromFolder(String fileFolderPath)
+  {
+    println("Files found: ");
+    filenames = listFileNames(fileFolderPath);
+    printArray(filenames);
+  }
+  
+  //read all game files from the folder the reader was created with
+  Path[] readGameFiles(String folder)
+  {
+    Path[] playerPaths = new Path[filenames.length];
+    for(int i = 0; i < filenames.length; i++)
+    {
+        playerPaths[i] = readTestFile(folder + "/" + filenames[i]);
+    }
+    return playerPaths;
+  }
+  
+  Path readTestFile(String filePath)
+  {
+    BufferedReader reader = createReader(filePath);
+    StringList lines = new StringList();
+    try
+    {
+      String line;
+      while((line = reader.readLine())!= null)
+      {
+        line = line.replace(',', '.');
+        lines.append(line);
+      }
+    }
+    catch (IOException e)
+    {
+      e.printStackTrace();
+    }
+    String[][] gameplayData = new String[lines.size()][8];
```

Comment:
magic number 😂

Reply:
Does it make more sense to have a comment here or to create a variable that we only use once? It is 8 because that is how many values we get in the file per line.

Reply:
Yeah that's true, but I'd say having a global variable like `dataElementAmount` or such would be better

Reply:
just thinking from the exam perspective if someone's asked about this piece of code, it's immediately clear where does it take that number from

Reply:
but we dont want to have a global variable that can be reached from anywhere too, do we?

Reply:
that's true, but it could then maybe be private in the class to just be more certain, what that number actually means